src/EditorFeatures/TestUtilities/Traits.cs

```diff
@@ -127,6 +127,7 @@ public static class Features
             public const string TypeInferenceService = "TypeInferenceService";
             public const string Venus = "Venus";
             public const string VsLanguageBlock = "VsLanguageBlock";
+            public const string VsNavInfo = "VsNavInfo";
```

Comment:
There's nothing to reference in a nameof. This is defined in the test utilities project, which is in the compiler layer.